cmd/depth/depth.go

```diff
@@ -73,12 +69,12 @@ func handlePkgs(t *depth.Tree, pkgs []string, outputJSON bool) error {
 	return nil
 }
 
-// writePkgSummary
+// writePkgSummary writes a summary of all packages in a tree
 func writePkgSummary(w io.Writer, pkg depth.Pkg) {
-	sum := newSummary()
+	sum := &summary{0,0,0}
```

Comment:
Last thing and it's a nitpick, but can you just initialize as `var sum summary` and use `&sum` below? It's more apparent that we're just initializing the zero-value summary, and passing a pointer to the method